pallets/token-mux/src/tests.rs

```diff
@@ -506,3 +506,61 @@ pub(crate) mod swaps {
 		})
 	}
 }
+
+mod is_local_representation {
+	use cfg_traits::HasLocalAssetRepresentation;
+	use cfg_types::tokens::CurrencyId;
+	use sp_runtime::DispatchError;
+
+	use super::*;
+	use crate::mock::{new_test_ext_invalid_assets, MockRegistry, USDC_WRONG_DECIMALS};
+
+	#[test]
+	fn is_local_happy_paths() {
+		new_test_ext().execute_with(|| {
+			assert_eq!(<CurrencyId as HasLocalAssetRepresentation<MockRegistry>>::is_local_representation_of(&USDC_LOCAL, &USDC_1), Ok(true));
```

Comment:
Just curious,

```rust
CurrencyId::is_local_representation_of()
```

Does not compile? Does the compiler force you to specify the trait?

Reply:
Unfortunately not, the `AssetInspect` parameter type cannot be inferred. Tried different variants but none of the shortcuts worked.